tests/rptest/tests/raft_availability_test.py

```diff
@@ -270,6 +282,14 @@ def test_two_nodes_down(self):
         # Bring back one node (not the original leader)
         self.redpanda.start_node(self.redpanda.get_node(other_node_id))
 
+        nodes = admin.get_redpanda_nodes()
+        nodes = [node for node in nodes if node.id != initial_leader_id]
+        admin.wait_stable_configuration("id_allocator",
+                                        namespace="kafka_internal",
+                                        replication=3,
+                                        timeout_s=ELECTION_TIMEOUT * 2,
```

Comment:
I missed a detail here on the first review (sorry).  The time up to ELECTION_TIMEOUT * 2 that we wait for the id_allocator topic is in addition to the time that we wait below for the test's topic to become available.  This is relaxing the success condition for the test: now the test topic could take up to `ELECTION_TIMEOUT * 4` to become available, which risks us missing raft bugs that prevent prompt elections.

Maybe `wait_stable_configuration` could return the time it consumed, and then subtract that from the timeout for the subsequent wait?  

(same for other tests in this class that are doing a similar routine)

Reply:
ci seems pretty noisy so in general I'm a bit reluctant to using it to measure performance and have strict timing assumption but here we have two raft groups which go through reelection, each group's reelection is capped by `ELECTION_TIMEOUT * 2` so if we there is a bug  in raft which cause election go longer than that we'll still catch it with same accuracy